build.gradle

```diff
@@ -35,7 +35,7 @@ android {
         // versionName for legacy version - placeholder needs to be set before building on CI
         versionName "2022.01.01-legacy"
         minSdkVersion 16
-        maxSdkVersion 19
+        maxSdkVersion 20
```

Comment:
Up to my knowledge API 20 is exclusively used for the first Android Wear OS. So, its not needed but should not harm either.

Reply:
Oh, I forgot this (or did not know before? :-) ).
https://developer.android.com/studio/releases/platforms#4.4

But I think it is easier if we define the version numbers of our releases witout a gap.